backend/src/test/java/com/ody/notification/service/NotificationServiceTest.java

```diff
@@ -176,6 +175,6 @@ void sendSendNudgeMessageMessage() {
 
         notificationService.sendNudgeMessage(requestMate, nudgedMate);
 
-        Mockito.verify(getFcmPushSender(), times(1)).sendNudgeMessage(any(), any());
+        Mockito.verify(fcmPushSender, Mockito.times(1)).sendNudgeMessage(any(), any());
```

Comment:
[제안] static 임포트 추가해주고 Mockito는 제거해도 괜찮을 것 같네요 😃

```suggestion
        Mockito.verify(fcmPushSender, times(1)).sendNudgeMessage(any(), any());
```

Reply:
Static import를 가급적 사용하지 않기로 얘기했던 것 같아서 일부로 추가하긴 했어요
 전체적으로 Static 메서드를 사용할 때 클래스를 명시되어 있어서 맞추고자 일부로 추가했습니다 !